### ISSUE TITLE
Perl RCs are not anymore at CPAN

### ISSUE THREAD

**Issue by eserte** [tokuhirom/Perl-Build#53]
Test suite recently started to fail:
```
WARN: [cpan_perl_releases] not found the tarball for perl-5.18.0-RC4
WARN: [perl_releases_page] not found the tarball for perl-5.18.0-RC4
WARN: [search_cpan_org] Cannot get content from http://search.cpan.org/dist/perl-5.18.0-RC4: 410 Gone
ERROR: Cannot find the tarball for perl-5.18.0-RC4
    # Child (CPAN::Perl::Releases) exited without calling finalize()

#   Failed test 'CPAN::Perl::Releases'
#   at /usr/perl5.8.9/lib/5.8.9/Test/Builder.pm line 279.
# Tests were run but no plan was declared and done_testing() was not seen.
# Looks like your test exited with 255 just after 1.
t/02_perl_release.t ...... 
Dubious, test returned 255 (wstat 65280, 0xff00)
Failed 1/1 subtests 
```

See also http://www.nntp.perl.org/group/perl.perl5.porters/2016/01/msg233742.html for the announcing of removal of RC versions by rjbs.

**Comment by tokuhirom** [comment]
hmm...
http://www.nntp.perl.org/group/perl.perl5.porters/2016/01/msg233715.html

**Comment by miyagawa** [comment]
just change the version to newer RC version or use backpan as a mirror base.

**Comment by miyagawa** [comment]
closed by #54
